src/app/pages/AddDestination/AddDestination.tsx

```diff
@@ -63,7 +64,7 @@ export default function AddDestination(): JSX.Element {
           onChange={(event) => setEndDate(event.target.value)}
         />
       </EndTrip>
-      <button>Go!</button>
+      <SubmitButton children="Go" />
```

Comment:
If you want to style this `SubmitButton` like ` grid-column: 3/5;`, you can create a styled version:

Solution 1:

```tsx
  const StyledSubmitButton = styled(SubmitButton)`
    grid-column: 3/5;
  `;
```

```tsx
<StyledSubmitButton>Go</StyledSubmitButton/>
```

Solution 2: (I prefer this)

```tsx
const Destination = styled.form`
  list-style: none;
  max-width: 1400;
  margin-left: auto;
  margin-right: auto;
  display: grid;
  grid-template-columns: repeat(6, 1fr);
  column-gap: 5px;
  border: black solid 1px;
  border-radius: 5px;
  justify-items: center;
  text-align: center;
  
  ${SubmitButton} {
    grid-column: 3/5;
  }
`;
```